src/clients/scc_auto.rb

```diff
@@ -212,6 +223,35 @@ def write
       true
     end
 
+    # Select the product from the control.xml (for the online installation medium)
+    # @return [Boolean] true on success, false on failure
+    def online_medium_config
+      # import the GPG keys before refreshing the repositories
+      Yast::Packages.ImportGPGKeys
+
+      products = Y2Packager::ProductControlProduct.products
+      ay_product = Profile.current.fetch("software", {}).fetch("products", []).first
+
+      if !ay_product
+        # TRANSLATORS: error message, %s is the XML path, e.g. "software/products"
+        Report.Error(_("Missing product specification in the %s section") % "software/products")
+        return false
```

Comment:
I prefer if specially designed class for autoyast issues will be used. I think this one with fatal severity is suitable https://github.com/yast/yast-autoinstallation/blob/master/src/lib/autoinstall/autoinst_issues/missing_value.rb

Reply:
Unfortunately the AY side expects only booleans [here](https://github.com/yast/yast-autoinstallation/blob/master/src/lib/autoinstall/autosetup_helpers.rb#L88-L95). Supporting custom AY issue would require bigger changes.

I'd keep it simple for now.